cumulus/parachains/runtimes/assets/asset-hub-kusama/src/weights/pallet_asset_conversion.rs

```diff
@@ -66,22 +62,22 @@ impl<T: frame_system::Config> pallet_asset_conversion::WeightInfo for WeightInfo
 	/// Proof: `PoolAssets::Account` (`max_values`: None, `max_size`: Some(134), added: 2609, mode: `MaxEncodedLen`)
 	fn create_pool() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `480`
-		//  Estimated: `6196`
-		// Minimum execution time: 88_484_000 picoseconds.
-		Weight::from_parts(92_964_000, 0)
-			.saturating_add(Weight::from_parts(0, 6196))
-			.saturating_add(T::DbWeight::get().reads(9))
+		//  Measured:  `408`
+		//  Estimated: `4689`
+		// Minimum execution time: 905_000_000 picoseconds.
+		Weight::from_parts(928_000_000, 0)
```

Comment:
Any idea why this went up ten times?

Reply:
probably my local machine, and target is `debug` instead `production`, you can see it in the header